examples/batches/scan_form_batch/scan_form_batch.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+    "encoding/json"
+    "fmt"
+    "github.com/EasyPost/easypost-go"
+    "os"
+)
+
+func main() {
+    apiKey := os.Getenv("EASYPOST_API_KEY")
+    if apiKey == "" {
+        fmt.Fprintln(os.Stderr, "missing API key")
+        os.Exit(1)
+        return
+    }
+    client := easypost.New(apiKey)
+
+    // ScanForm Batch
+    batch, err := client.CreateBatchScanForms(
+        "batch_12345",
+        "string",
```

Comment:
Let's delete this, if instead you wanted to show that this could be another batch, let's do that instead with something like `batch_456`, the above batch: `batch_12345` also could better follow other examples and truncate at the third digit.